test/e2e/apps/replica_set.go

```diff
@@ -35,7 +36,37 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *extensions.ReplicaSet {
+func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *apps.ReplicaSet {
```

Comment:
Just chatted with @kow3ns and he had a good point: we should keep the e2e tests unchanged and if this PR is doing the right changes, the e2e tests using `extensions/vebeta1` should still work. The e2e tests can be updated when we are ready to update kubectl.